web/app.js

```diff
@@ -2626,9 +2625,9 @@ function webViewerWheel(evt) {
       // left corner is restored. When the mouse wheel is used, the position
       // under the cursor should be restored instead.
       const scaleCorrectionFactor = currentScale / previousScale - 1;
-      const rect = pdfViewer.container.getBoundingClientRect();
-      const dx = evt.clientX - rect.left;
-      const dy = evt.clientY - rect.top;
+      const [top, left] = pdfViewer.containerTopLeft;
```

Comment:
In e.g. a custom deployment of the viewer, are these top and left values *guaranteed* to never change after the viewer has been initialized?

Reply:
It'd be strange to change the position but why not...